firebase.json

```diff
@@ -268,6 +268,31 @@
         "source": "/docs/decentralized-oracles-ethereum-mainnet",
         "destination": "/docs/any-api-testnet-nodes/",
         "type": 301
+      },
+      {
+        "source": "/docs/get-a-random-number/",
+        "destination": "/docs/vrf/v2/subscription/get-a-random-number/",
```

Comment:
What's the rationale for adding `/v2/`? Does it just make it easier to know which version of the docs you are reading?

Reply:
We need some structure there mainly for scalability in the future:

- What happens if we have different versions in v3, v4...ETc
- Naming becomes much more easier: For every version we have  /introduction , /configuration, /security , /best-practises ..Etc